Ocelot.Downstream.Service.Instance2/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace Ocelot.Downstream.Service.Instance2.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+    };
+
+        private readonly ILogger<WeatherForecastController> _logger;
+
+        public WeatherForecastController(ILogger<WeatherForecastController> logger)
+        {
+            _logger = logger;
+        }
+
+        [HttpGet("hostname")]
+        public string GetHostName()
+        {
+            return Environment.MachineName + "Instance2";
+        }
+
+        [HttpGet(Name = "GetWeatherForecast")]
+        public IEnumerable<WeatherForecast> Get()
```

Comment:
You don't need extra code, if you want to show this, then provide the ocelot.json accordingly

Reply:
Removed this API